_build/sharpmake/src/main.cs

```diff
@@ -184,7 +184,7 @@ private static void InitializeGenerationSettings(Dictionary<string, ConfigSettin
 
     ProjectGen.Settings.ClangTidyRegex = config["clang-tidy-regex"].Value.GetString();
     ProjectGen.Settings.PerformAllClangTidyChecks = config["perform-all-clang-tidy-checks"].Value.GetBoolean();
-    ProjectGen.Settings.NoClangTools = config["no-clang-tools"].Value.GetBoolean();
+    ProjectGen.Settings.ClangToolsEnabled = config["no-clang-tools"].Value.GetBoolean() == false;
```

Comment:
INFO: Is there a reason why this remains "no-clang-tools" in the config file?

Reply:
Would we have to add this flag to disable the clang tools every time we request a new generation cuz then I rather not but otherwise I would make it consistent with the rest

Reply:
I'll disable clang tools by default, that'll make things easier

Reply:
done